### PR TITLE
added homepage

### PR DESCRIPTION
**Pull request by Chiragmodi01**
image

### CODE REVIEW

File: Index.html [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+                    <div class="trendingHer_img-overlay">
+                      <p class="trendingHer_img_overlay_product-desc">This is product desc</p>
+                      <p class="trendingHer_img_overlay_brand-name">Brand</p>
+                    </div>
+                  <img src="https://picsum.photos/id/129/550/300/?blur=1" alt="img" class="trendingHer_section-img" />
+                  </div>
+              </div>
+              <div class="homePage_trendingHer_section section-2">
+                  <div class="trendingHer_section_item item-1">
+                    <div class="trendingHer_img-overlay">
```
Comment (amirnathani9) [inline Index.html:94]: here is the suggestion giving the class name in one convention

**Review by amirnathani9 — APPROVED** [review]
Code is clean and good to merge.
Suggestions are to use the relative path convention for the file paths.

**Review by rahul31agrawal — APPROVED** [review]
looks good to merge

**Review by aurobindoGupta — APPROVED** [review]
looks good

**Pull request merged**